build_projects/dotnet-host-build/PackageTargets.cs

```diff
@@ -266,5 +269,25 @@ private static void FixPermissions(string directory)
                 FS.FixModeFlags(directory);
             }
         }
+
+        private static string GetSharedFrameworkPublishPath()
```

Comment:
CCing @gkhanna79 @Petermarcu  to provide more details regarding the priority . As of now, I am targeting 4/14 (EOW) to get this change in.